lib/zendesk_apps_tools/common.rb

```diff
@@ -10,6 +41,10 @@ def say_error(msg)
     end
 
     def get_value_from_stdin(prompt, opts = {})
+      if options[:unattended]
```

Comment:
That would be ok, mind if we go with a positive option rather than a negative option?